resources/rpi/user-data

```diff
@@ -0,0 +1,74 @@
+#cloud-config
+# https://cloudinit.readthedocs.io/
+
+# Set the first part of the name to camera device.
+hostname: huntsmanpi000
+
+# If you have set up ssh on github you can pull down your
+# key automatically so that you can log into the unit without
+# a password. Change 'CHANGEME' to your username below.
+ssh_import_id:
+  - gh:huntsmanscope
+
+chpasswd:
+  expire: true
```

Comment:
Is this enabling password expiry?

Reply:
It causes it to expire immediately so you are forced to change the password on first login.  I'll add a comment.